app/models/analyticEvent.server.test.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  getTallyEvents,
+  tallyAnalytics,
+  addAnalyticEvent,
+  getRecentEvents,
+  deleteOldEvents
+} from "./analyticEvent.server";
+
+test("analytic event server: test mocked db part1 tally events", async () => {
+  const intialVal = await getTallyEvents();
+  expect(intialVal).toEqual([]);
+
+  const dateUnixTimeStamps: number[] = [
+    678293369, 678293369, 678293369, 678293369, 778293369, 778293369, 778293369
+  ];
+
+  for (const ts of dateUnixTimeStamps) {
+    addAnalyticEvent("testIp", "testEvent", "testData", ts);
+  }
+
+  const recentEvents = await getRecentEvents();
+
+  await tallyAnalytics();
+  const tallyEvents = await getTallyEvents();
+  expect(tallyEvents[0].count).toEqual(4);
```

Comment:
expect:
that there are only 2 tallyEvents
that the day of the tallyEvents match what we expect from dayX, dayY above

Reply:
Resolved.